processor.py

```diff
@@ -129,9 +128,9 @@ def get_parameters(self):
         """gets from all wrapped processors"""
         d = {}
         for p in self.processors:
-            parameter_names = p.PARAMETERS.keys()
+            parameter_names = list(p.PARAMETERS.keys())
             parameter_values = [getattr(p, n) for n in parameter_names]
-            d.update(dict(zip(parameter_names, parameter_values)))
+            d.update(dict(list(zip(parameter_names, parameter_values))))
```

Comment:
Good point. This was added by `2to3`, and I removed a lot of unnecessary `list` calls, but clearly I missed this one.